release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2477 @@
+users to discover and instantiate those services for use with their
+applications.
+
+As a result, service users benefit from ease and consistency of use across
+different types of services from different providers, while service providers
+benefit from having one integration point that gives them access to multiple
+platforms.
+
+[IMPORTANT]
+====
+This feature is not for production workloads and is currently not fully
+supported. This is using an alpha version of the Kubernetes service catalog.
+====
+
+This feature consists of:
+
+- The Service Consumer: The individual, application , or service that uses a service enabled by the broker and catalog.
+- The Catalog: Where services are published for consumption.
+- Service Broker: Publishes services and intermediates service creation and credential configuration with a provider.
+- Service Provider: The technology delivering the service.
+- Open Service Broker API: Lists services, provisions and deprovisions, binds, and unbinds.
+
+See
+xref:../install_config/install/advanced_install.adoc#enabling-service-catalog[Enabling
+the Service Catalog] for more information.
+
+[[ocp-36-ansible-service-broker]]
+==== Ansible Service Broker (Technology Preview)
+
+- Implementation of Open Service Broker API that enables users to leverage Ansible
+for provisioning and managing of services via the Service Catalog on OpenShift.
+- Standardized approach for delivering simple to complex multi-container
+OpenShift services.
+- Works in conjunction with Ansible Playbook Bundles (APB), which is a lightweight
+meta container comprised of a few named playbooks for each Open Service Broker
+API operations.
+
+Service Catalog and ASB must be configured during OpenShift installation. Once
+enabled, APB services can be deployed right from Service Catalog UI.
+
+See
+xref:../install_config/install/advanced_install.adoc#configuring-ansible-service-broker[Configuring
+the Ansible Service Broker] for more information.
+
+[[ocp-36-template-service-broker]]
+==== Template Service Broker (Technology Preview)
+
+The Template Service Broker (currently in
+xref:ocp-36-technology-preview[Technology Preview]) exposes OpenShift templates
+through a Open Service Broker API to the Service Catalog.
+
+The Template Broker matches the lifecycles of provision, deprovision, bind,
+unbind with existing templates. No changes are required to templates, unless you
+expose bind. Your application will get injected with configuration details
+(bind).
+
+[WARNING]
+====
+The template service broker (TSB) is currently a Technology Preview feature and
+should not be used in production clusters. Enabling the TSB currently requires
+opening unauthenticated access to the cluster; this security issue will be
+resolved before exiting the Technology Preview phase.
+====
+
+See
+xref:../install_config/install/advanced_install.adoc#configuring-template-service-broker[Configuring
+the Template Service Broker] for more information.
```

Comment:
The content for `Template Service Broker (Technology Preview)` is OK.
But seems it does not choose good parent section `Web Console`.
It is better to put it under `Installation` or `Developer Experience`. The "template service broker" feature card https://trello.com/c/2FY9BDpS/1204-5-template-broker-configuration-templatebroker is in `Developer Experience` board
@wzheng1 please confirm
@yapei just FYI above